apps/back/migrations/Version20230906111240.php

```diff
@@ -7,26 +7,21 @@
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\Migrations\AbstractMigration;
 
-/**
- * Auto-generated Migration: Please modify to your needs!
- */
-final class Version20230905083639 extends AbstractMigration
+final class Version20230906111240 extends AbstractMigration
 {
     public function getDescription(): string
     {
-        return '';
+        return 'Database main for knp-taste application';
```

Comment:
Description is used to define the purpose of the migration. On this case, this PR will add courses and user tables